sct-commons/src/test/resources/scd-ldepf/scd_ldepf_dataTypeTemplateValid.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!-- SPDX-FileCopyrightText: 2023 RTE FRANCE -->
+<!-- -->
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+<SCL version="2007" revision="B" release="4" xmlns="http://www.iec.ch/61850/2003/SCL" xmlns:compas="https://www.lfenergy.org/compas/extension/v1">
+    <Private type="COMPAS-SclFileType">
+        <compas:SclFileType>SCD</compas:SclFileType>
+    </Private>
+    <Header id="hId" version="2007" revision="B" toolID="COMPAS"/>
+    <IED name="IEDTEST">
+        <AccessPoint name="AP_NAME"/>
+    </IED>
+    <IED name="IED_NAME1">
```

Comment:
check necessity of some comments as the informations clear

Reply:
unnecessary comments are removed